osm2streets/src/render.rs

```diff
@@ -268,6 +268,69 @@ impl StreetNetwork {
         Ok(output)
     }
 
+    /// Generates polygons that cover the entire road area. One polygon per Road and Intersection.
```

Comment:
A pretty quick refactor that might move us forward is just returning `geojson::GeoJson` from all the existing methods. The WASM API can then to `serde_json::to_string_pretty` and the Java can extract the feature collection

Reply:
That was the idea I've been flirting with. I think it's worth writing types for the different layers of output we want to return though, and optionally turn that into GeoJSON as an optional output format step.

I am valuing a principled render-output API (same idea as the "semantic lane markings" idea), and the types can serve as documentation for the GeoJSON that we generate too.

Reply:
That would be even nicer, yeah! #24 related. I wouldn't block this JOSM integration with that, but certainly would love to see this change happen